docs: update self-hosted tutorial for self-hosting analytics server (#15755)
supabase/supabase: apps/docs/pages/guides/self-hosting/docker.mdx

By default, Storage backend is set to `file`, which is to use local files as the storage backend. To make it work on macOS, you need to choose `VirtioFS` as the Docker container file sharing implementation (in Docker Desktop -> Preferences -> General).

### Setting up logging with the Analytics server

> Additional configuration is required for self-hosting the Analytics server. Full setup instructions can be found [here](https://supabase.com/docs/reference/self-hosting-analytics/introduction#getting-started).

Reviewer: ```suggestion
Additional configuration is required for self-hosting the Analytics server. For the full setup instructions, see [Self Hosting Analytics](https://supabase.com/docs/reference/self-hosting-analytics/introduction#getting-started).
```
